plugins/webexDropFolderPlugin/batch/KWebexDropFolderEngine.php

```diff
@@ -228,15 +242,24 @@ protected function purgeFiles ()
 	}
 
 	/**
-	 * @param string $fileCreateTime
+	 * @param string $fileName
 	 * @throws Exception
 	 */
-	private function deleteFileFromRecycleBin($fileCreateTime)
+	private function deleteFileFromRecycleBin($fileName)
```

Comment:
What about deleting from recycle bin based on ID? In place that we have ID it is better to delete by it.